pkg/addons/data/ClusterRole.rbac.authorization.k8s.io/cluster-monitoring-operator.yaml

```diff
@@ -0,0 +1,246 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
This role should be reviewed, consolidated, and verified.  There are some permissions in here that could potentially grant root on a cluster.